docs/services/navigation/_index.md

```diff
@@ -15,19 +15,20 @@ The Navigation service is the stateful definition of Viam's [motion service](/se
 It uses GPS to autonomously navigate a rover [base](/components/base/) to user defined endpoints called waypoints.
 Configure your base with a navigation service, add waypoints, and set the mode of the service to [**Waypoint**](#setmode) to move your rover along a defined path at your desired motion configuration.
 
-## Configuration
+## Requirements
 
-You must configure a [base](/components/base/) with a [movement sensor](/components/movement-sensor/) as part of your robot to configure a Navigation service.
+You must configure a [base](/components/base/) with [movement sensors](/components/movement-sensor/) as part of your robot to configure a Navigation service.
 
-{{% alert title="Important" color="note" %}}
+Configure a stack of movement sensors that satisfy the following requirements in their {{< glossary_tooltip term_id="model" text="models'" >}} implementations of the [Movement Sensor API](/components/movement-sensor/#api):
 
-Make sure the [movement sensor](/components/movement-sensor/) you use supports [`GetPosition()`](/components/movement-sensor/#getposition) and at least one of [`GetCompassHeading()`](/components/movement-sensor/#getcompassheading) or [`GetOrientation()`](/components/movement-sensor/#getorientation) in its {{< glossary_tooltip term_id="model" text="model's" >}} implementation of the [Movement Sensor API](/components/movement-sensor/#api).
+- [`GetPosition()`](/components/movement-sensor/#getposition)
+- [`GetAngularVelocity()`](/components/movement-sensor/#getangularvelocity)
+- [`GetLinearVelocity()`](/components/movement-sensor/#getlinearvelocity)
+- At least one of [`GetCompassHeading()`](/components/movement-sensor/#getcompassheading) or [`GetOrientation()`](/components/movement-sensor/#getorientation). If your movement sensor stack provides multiple methods, your robot will default to using the values returned by `GetCompassHeading()`.
```

Comment:
```suggestion
- At least one of [`GetCompassHeading()`](/components/movement-sensor/#getcompassheading) or [`GetOrientation()`](/components/movement-sensor/#getorientation). If your movement sensor stack provides multiple methods, your robot will default to using the values returned by `GetCompassHeading()`.
```
@raybjork or @Fahmina -- is this still accurate?

Reply:
I think technically this is still accurate but we will be deprecating this in the future.  From a documentation perspective the only thing we should be advertising to users is that CompassHeading is the necessary function.

Reply:
Ok, sounds good! Updating